performance.rst

```diff
@@ -74,6 +75,14 @@ container into a single file, which could improve performance when using
         // ...
         $container->setParameter('container.dumper.inline_factories', true);
 
+.. _performance-enabled-locales:
+
+Restrict the number of locales enabled in the application
```

Comment:
Should it be title case here?

Reply:
Please, add it yourself (`Restrict the Number of Locales Enabled in the Application`) because I tried to do this myself but I can't (unless I destroy your commit 😢  ... and i don't want to do that). Thanks.

Reply:
I don't care about authorship :), just squash away, thanks!

Reply:
There was also a typo in the ref link and I was able to squash locally. Thanks